src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -301,7 +300,7 @@ public override void VisitMethod(IMethodSymbol symbol)
             // symbol then we do not know its accessibility, modifiers, etc, all of which require knowing
             // the containing type, so we'll skip them.
 
-            if ((object)symbol.ContainingType != null || (symbol.ContainingSymbol is ITypeSymbol))
+            if ((object?)symbol.ContainingType != null || (symbol.ContainingSymbol is ITypeSymbol))
```

Comment:
```suggestion
            if (symbol.ContainingType is not null || (symbol.ContainingSymbol is ITypeSymbol))
```

Reply:
➡️ I left this for a separate refactoring.